packages/bundle-source/package.json

```
@@ -39,7 +39,7 @@
     "@rollup/plugin-node-resolve": "^13.0.0",
     "acorn": "^8.2.4",
     "jessie.js": "^0.3.2",
-    "rollup": "endojs/endo#rollup-2.7.1-patch-1",
+    "rollup": "^2.77.4-1",
```

It's a little strange to leave the `-1` suffix.  Is there a reason why you did that?

```suggestion
    "rollup": "^2.77.4",
```

---

It was the latest version according to whatever I looked up, but the yarn.lock shows a newer version so I'll change this to that.